ui/shared/status/StatusEmojiPopup.qml

```diff
@@ -8,11 +8,15 @@ import "../../shared"
 import "./emojiList.js" as EmojiJSON
 
 Popup {
+    id: popup
     property var emojiSelected: function () {}
     property var categories: []
     property alias searchString: searchBox.text
+    property string skinColor: ""
```

Comment:
Instead of keeping this value as a prop, we should put it in the QtSettings here: https://github.com/status-im/status-desktop/blob/master/ui/app/AppMain.qml#L348

That way, the user wouldn't have to re-set their skin color each time.

Reply:
done